html_app/ui/FacsView.js

```diff
@@ -799,8 +811,8 @@ scb.ui.static.FacsView.evaluate_chart = function (state) {
                 if (!isNaN(from)) {
                     var to = px;
                     to = to > 0 ? to : 0;
-                    to = to > scb.ui.static.FacsView.MAX_GATE  ? scb.ui.static.FacsView.MAX_GATE  : to;
-                    to = to < 0 ? 0 : to;
+                    to = to > max_x ? max_x  : to;
+                    to = to < 0 ? 0 : to; //not sure why this is needed?
```

Comment:
this is really terse way to do multiple logic branches and assign value.  It would be more clear to use an explicit if(){}else if{}.... structures to assign `to` only one time.